db/schema.rb

```diff
@@ -49,6 +50,16 @@
   add_index "admin_users", ["email"], name: "index_admin_users_on_email", unique: true, using: :btree
   add_index "admin_users", ["reset_password_token"], name: "index_admin_users_on_reset_password_token", unique: true, using: :btree
 
+  create_table "applications", force: :cascade do |t|
+    t.string   "title"
+    t.string   "url"
+    t.string   "provider"
+    t.string   "app_id"
+    t.integer  "integration_id"
+    t.datetime "created_at",     null: false
+    t.datetime "updated_at",     null: false
+  end
```

Comment:
Why do we still have the applications table? You need to update the schema and remove this chunck of code if you removed the migration and also drop the table from your db manually. Every time you make a migration and you run it and then we decide to remove that you need to run `rake db:rollback` and then remove the migration file.